src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -29,6 +29,8 @@ public static class BridgeClientCertificateManager
         private const string ClientCertificateSubject = "WCF Client Certificate";
         private const string ClientCertificatePassword = "test"; // this needs to be kept in sync with the Bridge configuration 
 
+        public static string Thumbprint { get; private set; }
```

Comment:
Nit: it would be good for a more descriptive name or a comment to describe what this is.  Could more than one test attempt to set this to different values?